src/winforms/toga_winforms/window.py

```diff
@@ -45,6 +45,10 @@ def create(self):
         self.toolbar_native = None
         self.toolbar_items = None
 
+        # Tell Winforms to heed self.native.Location on window creation
+        self.native.StartPosition = WinForms.FormStartPosition.Manual
+        self.native.Location = Point(*self.interface.position)
```

Comment:
`self.interface.position` is determined by invoking `get_position` - so you'll be setting the initial position of the window to the position that window is currently in. The position provided by the user at the time of construction will be lost. This is a bootstrapping problem.

Reply:
hmm...I see what you mean, but it **does** work: the position provided from the user is **not** lost and the window appears where it should (see the window.py example)